pkg/fileutils/read.go

```diff
@@ -88,7 +97,7 @@ func readAbsFile(file string) ([]byte, error) {
 
 func downloadFile(url string) ([]byte, error) {
 	t := &http.Transport{}
-	t.RegisterProtocol("file", http.NewFileTransport(http.Dir("/")))
+	t.RegisterProtocol("file", http.NewFileTransport(http.Dir("/c/")))
```

Comment:
Why is the `Dir` being created with a directoty tree of **"/c/"** ?

Reply:
It was temporary code. I'll investigate.

Reply:
Removed dead code.